backends/openweathermap.org.go

```diff
@@ -82,15 +93,52 @@ func (c *openWeatherConfig) fetch(url string) (*openWeatherResponse, error) {
 	if c.debug {
 		fmt.Printf("Response (%s):\n%s\n", url, string(body))
 	}
+	if res.StatusCode != 200 {
+		err = openWeatherErrorReponseHandler(body, url)
+		return nil, err
+	} else {
```

Comment:
No need for this `else`, because you have a return above.

Reply:
I have removed the else block and handled the API response outside of the else block